src/test/e2e/21_logging_test.go

```diff
@@ -12,17 +14,20 @@ func TestLogginga(t *testing.T) {
 	e2e.setup(t)
 	defer e2e.teardown(t)
 
+	// Get a random local port for this instance
+	localPort, _ := k8s.GetAvailablePort()
+
 	// Establish the port-forward into the logging service
-	err := e2e.execZarfBackgroundCommand("connect", "logging", "--cli-only")
+	err := e2e.execZarfBackgroundCommand("connect", "logging", fmt.Sprintf("--local-port=%d", localPort), "--cli-only")
```

Comment:
Can we have one of the tests not provide the `--local-port` flag for the 'default` connection. 

The tunnel logic is now handling that and dynamically using an open port and I want to make sure we are testing that functionality too.